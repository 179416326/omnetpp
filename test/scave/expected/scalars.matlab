scalars.description="scalar desc"
scalars.File=[
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
]

scalars.Run=[
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
"";
]

scalars.Module=[
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
]

scalars.frames_dropped=[
0;
NaN;
NaN;
0;
0;
0.002;
0;
0;
0.012;
0;
0.009;
0.014;
0.021;
0.022;
0.026;
0.051;
0.092;
0.144;
0.254;
0.434;
]

scalars.offered_load=[
30;
40;
10;
20;
50;
NaN;
70;
80;
90;
100;
110;
120;
130;
140;
150;
160;
170;
180;
190;
200;
]

scalars.throughput=[
0.3511;
0.4506;
0.0812;
0.2296;
0.5321;
0.5988;
NaN;
0.6981;
0.7347;
0.7646;
0.7891;
0.8092;
0.8257;
0.8391;
0.845;
0.8392;
0.8086;
0.7487;
0.6877;
0.5716;
]

