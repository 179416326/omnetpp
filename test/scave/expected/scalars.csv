File,Run,Module,frames dropped,offered load,throughput
testfiles/scalars.sca,,Net.subnet[0],0,30,0.3511
testfiles/scalars.sca,,Net.subnet[0],NaN,40,0.4506
testfiles/scalars.sca,,Net.subnet[0],NaN,10,0.0812
testfiles/scalars.sca,,Net.subnet[0],0,20,0.2296
testfiles/scalars.sca,,Net.subnet[0],0,50,0.5321
testfiles/scalars.sca,,Net.subnet[0],0.002,NaN,0.5988
testfiles/scalars.sca,,Net.subnet[0],0,70,NaN
testfiles/scalars.sca,,Net.subnet[0],0,80,0.6981
testfiles/scalars.sca,,Net.subnet[0],0.012,90,0.7347
testfiles/scalars.sca,,Net.subnet[0],0,100,0.7646
testfiles/scalars.sca,,Net.subnet[0],0.009,110,0.7891
testfiles/scalars.sca,,Net.subnet[0],0.014,120,0.8092
testfiles/scalars.sca,,Net.subnet[0],0.021,130,0.8257
testfiles/scalars.sca,,Net.subnet[0],0.022,140,0.8391
testfiles/scalars.sca,,Net.subnet[0],0.026,150,0.845
testfiles/scalars.sca,,Net.subnet[0],0.051,160,0.8392
testfiles/scalars.sca,,Net.subnet[0],0.092,170,0.8086
testfiles/scalars.sca,,Net.subnet[0],0.144,180,0.7487
testfiles/scalars.sca,,Net.subnet[0],0.254,190,0.6877
testfiles/scalars.sca,,Net.subnet[0],0.434,200,0.5716
